src/app/personside/visittkort/header/status/Sivilstand.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
```

Comment:
kanskje litt heavy å lage en komponent for dette..?

hva med å bruke en const i rendermetoden til PersonStatus istedenfor? feks:

````
function PersonStatus({person}: PersonProps) {
    const sivilstand = `${person.sivilstand.beskrivelse} / 2 barn (under 21)`;
    return (
        <Undertekst>
            <NoWrap>
                <FødselsnummerLinje person={person}/>
                <PadLeft>
                    <Statsborgerskap statsborgerskap={person.statsborgerskap}/> / {sivilstand}
                </PadLeft>
            </NoWrap>
        </Undertekst>
    );
}
```

Reply:
Jepps, men tenker det kan være en grei diskusjon å ta, for det vil jo få litt å si for hvordan kodebasen ser ut fremover.

Tenker at formålet med abstraksjon er å gjøre koden enklere å lese og forvalte? I disse tilfellene opplever jeg at det gjør koden tyngre å lese. Min alternative løsning på to linjer vs din løsning med 2 ekstra filer og 28 koder linje. Da tenker jeg at løsningen med 2 filer og 28 koder linje må ha en merverdi, og den ser jeg ikke helt slik den er nå.

Da tenker jeg at det er en bedre strategi å gå for den enkleste og minste løsningen som løser problemet, og heller endre komponenten når den eventuelt blir for stor en gang.

Reply:
Bra med diskusjon :) Jeg synes fremdeles det er et bedre abstraksjonsnivå, lettere å lese, lettere å teste og forvalte med små komponenter vs. å samle det i en komponent.